pkgs/os-specific/darwin/sketchybar/default.nix

```
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchFromGitHub, Carbon, Cocoa, SkyLight }:
+
+let
+  target = {
+    "aarch64-darwin" = "arm";
+    "x86_64-darwin" = "x86";
+  }.${stdenv.hostPlatform.system};
```

```suggestion
  }.${stdenv.hostPlatform.system} or throw "Unsupported system: ${system}";
```

otherwise eval fails on linux.